src/CefBrowser/Runtime/WebBrowserClient.cs

```diff
@@ -203,7 +244,8 @@ internal void Init()
 				                                                 $"-javascript {javascript} " +
 				                                                 $"-bcr {backgroundColor.r} -bcg {backgroundColor.g} -bcb {backgroundColor.b} -bca {backgroundColor.a} " +
 				                                                 $"-log-path \"{logPath.FullName}\" -log-severity {logSeverity} {cachePathArgument}" +
-				                                                 $"-port {port} -debug {debugLog}")
+				                                                 $"{noProxyServerArgument} {mediaStreamArgument} {remoteDebuggingArgument} "+
```

Comment:
We should put the arguments that go directly to CEF as the last.
So it should look like:
```csharp

$"-port {port} -debug {debugLog} " +
$"{noProxyServerArgument} {mediaStreamArgument} {remoteDebuggingArgument}");														
```